src/panels/history/ha-panel-history.ts

```diff
@@ -265,6 +338,7 @@ class HaPanelHistory extends SubscribeMixin(LitElement) {
             changedProps.has("_areaDeviceLookup"))))
     ) {
       this._getHistory();
+      this._getStats();
```

Comment:
Assuming we call this as part of refreshing data, shouldn't we also have this be called when a user clicks the refresh button?

Reply:
I don't remember if I considered this in the initial pass, but wouldn't refreshing statistics just return the same data a second time? The statistics is not really live updating like the history is, so there's never any "new" data to get. So would we really want to bother querying and fetching it all again?

Reply:
So the one case where it's useful is the edge case where a user has the dashboard open permanently, on a period that relies on both statistics and states and hits refresh once a day without ever refreshing the page. So yeah, probably not a big deal.